Lib/test/test_buffer.py

```diff
@@ -2510,22 +2511,9 @@ def test_memoryview_sizeof(self):
         check(memoryview(a), vsize(base_struct + 3 * per_dim))
 
     def test_memoryview_struct_module(self):
-
-        class INT(object):
-            def __init__(self, val):
-                self.val = val
-            def __int__(self):
-                return self.val
-
-        class IDX(object):
-            def __init__(self, val):
-                self.val = val
-            def __index__(self):
-                return self.val
-
         def f(): return 7
 
-        values = [INT(9), IDX(9),
+        values = [FakeInt(9), FakeIndex(9),
```

Comment:
I don't quite understand the name, there's nothing really fake about these classes.  Also, it's nice to see what the classes do without leaving the file.

Reply:
I agree with both points here.

When scanning through the PR on GitHub and seeing `FakeInt(10.9)`, my first thought is that I don't know what that means without guessing, so I want to quickly find the `FakeInt` definition.  That's harder when that definition is in another file. (And yes, if you have a decent editor open and set up properly, the definition should be just a click or keystroke away, but not everyone has that setup available all of the time.)

The problem could potentially be mitigated with a name that more clearly describes what the class is doing (`HasDunderIntReturning(3.2)`?), but as you can tell I'm having trouble coming up with a name that's going to be immediately unambiguous for people reading the code without also being unwieldy.